source/layouts/blog.erb

```diff
@@ -2,9 +2,14 @@
 <% partial_for :sidebar, 'blog/sidebar' %>
 <% partial_for :foot, 'footer' %>
 <% wrap_layout :layout do %>
-  
+
   <article class="blog-post">
     <div class="blog-post-header">
+      <% if current_page.data.tags.include? 'Newsletter' %>
+        <a class="blog-header" href="<%= tag_path 'Newsletter' %>">
```

Comment:
Please add aria-label="read all blog posts marked newsletter" or something similar- right now this is showing up as a blank link in the accessibility tree. Thank you!

Reply:
Thank you for the heads up, I just read some more about it and I added the aria-label with a wording including `Ember Times` following a [suggestion on the WCAG working group Wiki to include a part of the original anchor text into the aria-label wording](https://www.w3.org/WAI/GL/wiki/Using_aria-label_for_link_purpose#Example_1:):

> [...] in that the simpler, non-descriptive [...] text on the page can be replaced with a more descriptive label of the link. The words [of the anchor text] are repeated in the aria-label (which replaces the original anchor text of "[Read more...]") to allow consistent communication between users.


<img width="1679" alt="screen shot 2018-07-03 at 01 04 44" src="https://user-images.githubusercontent.com/8811742/42190625-716b09d0-7e5e-11e8-97ab-f1232f6c6ca3.png">